introduction.md

```diff
@@ -0,0 +1 @@
+My name i carl nyden and i am quite new to github but i am quite quick to learning and will be quite usfull.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.